research/exisiting-landscape.md

```diff
@@ -0,0 +1,169 @@
+# Goal
+
+The goal of this research is to document the APIs and SDKs of existing feature flag
+management tools in order to identity patterns that should be considered when
+defining the OpenFeature spec.
+
+> NOTE: This research focuses primarily on the server-side NodeJS SDKs
```

Comment:
Add research status. Is it completed or a living document? How others could contribute? Any feedback needed? Maybe a `## Contribute` section in the beginning would be beneficial